Dfe.Academies.External.Web.UnitTest/Services/ConversionApplicationCreationServiceTests.cs

```diff
@@ -8,14 +9,17 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using AutoFixture;
 
 namespace Dfe.Academies.External.Web.UnitTest.Services;
 
 [Parallelizable(ParallelScope.All)]
-internal sealed class AcademisationCreationServiceTests
+internal sealed class ConversionApplicationCreationServiceTests
 {
+	private static readonly Fixture Fixture = new();
+
     [Test]
```

Comment:
Could this test attribute be commented out or removed until these tests are implemented? We don't want it to look like these scenarios are being tested when they're not.